### DISCUSSION TITLE
Is there a way to disable nuxt default cache-control strategy?

### DISCUSSION THREAD

**Discussion by Applelo** [nuxt/nuxt#27667] [Questions]
Hi everyone 👋  I am working on website and I was asked to remove the cache-control `"public, max-age=31536000, immutable"` on _nuxt/** files because we have a CDN and it can conflict with this header. 
However, after some search, I have found the header is hardcoded on Nuxt (see https://github.com/nuxt/nuxt/blob/a42a3869be7c5bbedad1d26f43812075d6dbb43f/packages/nuxt/src/core/nitro.ts#L134C11-L138C12) or maybe I miss something 🤔 
Any ideas is welcome 😄

**Comment by Applelo** [comment]
Ok so I have miss understand this is not the problem for the CDN. I reopen another discussion if necessary.

**Comment by censujiang** [comment]
same question

**Comment by jeannen** [comment]
Any solution for this? I need to change the default cache-control strategy because it conflicts with some of my setup
